### PR TITLE
Send all errors that occur in the install script

### PR DESCRIPTION
**Pull request by LiliC**
This also adds a reserved error code 111 to bootstrap, which is used to
exclude the errors, so we only send the only produced in the script
itself.

### CODE REVIEW

File: service/static/install.sh [hunk 1]
```diff
@@ -1,6 +1,24 @@
-#!/bin/sh
+#!/bin/bash
```
Comment (dlespiau) [inline service/static/install.sh]: We should be able to leave it ash `/sh`, don't seen any bashism.

File: service/static/install.sh [hunk 2]
```diff
@@ -20,19 +38,15 @@ curl -s >/dev/null 2>/dev/null -H "Accept: application/json" -H "Authorization:
...
-trap 'exit $?' 1 2 3 15
-
 # Get distribution
 unamestr=$(uname)
 if [ "$unamestr" = 'Darwin' ]; then
     dist='darwin'
 elif [ "$unamestr" = 'Linux' ]; then
     dist='linux'
+else
+  echo "This distribution is not supported"
```
Comment (dlespiau) [inline service/static/install.sh]: Maybe OS instead of distribution?

Comment (LiliC) [reply service/static/install.sh]: Done.

File: service/static/install.sh (hunk 1 above)
Comment (LiliC) [reply service/static/install.sh]: Done.

**Pull request merged**